pkg/report/testdata/linux/report/416

```diff
@@ -0,0 +1,65 @@
+TITLE: WARNING in usbhid_submit_report
+
+[ 1183.284526][T22234] WARNING: CPU: 1 PID: 22234 at mm/page_alloc.c:4696 __alloc_pages_nodemask+0x36f/0x780
+[ 1183.294469][T22234] Kernel panic - not syncing: panic_on_warn set ...
+[ 1183.301070][T22234] CPU: 1 PID: 22234 Comm: syz-executor.5 Not tainted 5.3.0-rc7+ #0
+[ 1183.308958][T22234] Hardware name: Google Google Compute Engine/Google Compute Engine, BIOS Google 01/01/2011
+[ 1183.319014][T22234] Call Trace:
+[ 1183.322317][T22234]  dump_stack+0xca/0x13e
+[ 1183.326562][T22234]  ? __alloc_pages_nodemask+0x320/0x780
+[ 1183.332129][T22234]  panic+0x2a3/0x6da
+[ 1183.336021][T22234]  ? add_taint.cold+0x16/0x16
+[ 1183.340769][T22234]  ? __probe_kernel_read+0x188/0x1d0
+[ 1183.346054][T22234]  ? __warn.cold+0x5/0x4a
+[ 1183.350384][T22234]  ? __alloc_pages_nodemask+0x36f/0x780
+[ 1183.355905][T22234]  __warn.cold+0x20/0x4a
+[ 1183.360146][T22234]  ? __alloc_pages_nodemask+0x36f/0x780
+[ 1183.365701][T22234]  report_bug+0x262/0x2a0
+[ 1183.370041][T22234]  do_error_trap+0x12b/0x1e0
+[ 1183.374646][T22234]  ? __alloc_pages_nodemask+0x36f/0x780
+[ 1183.380200][T22234]  do_invalid_op+0x32/0x40
+[ 1183.384619][T22234]  ? __alloc_pages_nodemask+0x36f/0x780
+[ 1183.390169][T22234]  invalid_op+0x23/0x30
+[ 1183.394328][T22234] RIP: 0010:__alloc_pages_nodemask+0x36f/0x780
+[ 1183.400484][T22234] Code: fe ff ff 65 48 8b 04 25 00 ef 01 00 48 05 60 10 00 00 41 be 01 00 00 00 48 89 44 24 58 e9 ee fd ff ff 81 e5 00 20 00 00 75 02 <0f> 0b 45 31 f6 e9 6b ff ff ff 8b 44 24 68 89 04 24 65 8b 2d e9 7e
+[ 1183.420083][T22234] RSP: 0018:ffff8881b13079d8 EFLAGS: 00010046
+[ 1183.426164][T22234] RAX: 0000000000000000 RBX: 1ffff11036260f3f RCX: 0000000000000000
+[ 1183.434112][T22234] RDX: 0000000000000000 RSI: dffffc0000000000 RDI: 0000000000040a20
+[ 1183.442059][T22234] RBP: 0000000000000000 R08: ffff8881ce986000 R09: ffffed103a0a9519
+[ 1183.450006][T22234] R10: ffffed103a0a9518 R11: ffff8881d054a8c7 R12: 0000000000000012
+[ 1183.457952][T22234] R13: 0000000000000012 R14: 0000000000000000 R15: ffff8881d054a8c0
+[ 1183.465924][T22234]  ? __alloc_pages_slowpath+0x2260/0x2260
+[ 1183.471620][T22234]  ? find_held_lock+0x2d/0x110
+[ 1183.476368][T22234]  alloc_pages_current+0xff/0x200
```

Comment:
What is it warning about? What is MAX_ORDER? I don't see any warnings referencing MAX_ORDER. Is it size? I think kmalloc should not warn on large requests...

Reply:
This is the warning that is getting triggered:

https://elixir.bootlin.com/linux/v5.3-rc7/source/mm/page_alloc.c#L4696

I don't know whether kmalloc should warn on large requests, but it does right not, unless `__GFP_NOWARN` is specified.

Reply:
Humm.. this is weird.
I fixed it. And we stopped getting warnings about large mallocs last November:
https://git.kernel.org/pub/scm/linux/kernel/git/davem/net.git/commit/?id=61448479a9f2
Is your setting slub/slab the same as for other instances? Please check if anything changed in kernel code. These warnings were a plague, if they are back it's bad.